doc/classes/TextServer.xml

```diff
@@ -1323,6 +1323,26 @@
 				Aligns shaped text to the given tab-stops.
 			</description>
 		</method>
+		<method name="string_to_lower" qualifiers="const">
+			<return type="String" />
+			<argument index="0" name="string" type="String" />
+			<argument index="1" name="language" type="String" default="&quot;&quot;" />
+			<description>
+				Returns the string converted to lowercase.
+				[b]Note:[/b] Casing is locale dependent and context sensitive if server support [code]FEATURE_CONTEXT_SENSITIVE_CASE_CONVERSION[/code] feature (supported by [TextServerAdvanced]).
```

Comment:
You can reference constants with `[constant FEATURE_CONTEXT_SENSITIVE_CASE_CONVERSION]`.

Reply:
Done.